api-contracts/openapi/paths/v1/webhooks/webhook.yaml

```diff
@@ -0,0 +1,264 @@
+V1WebhookGetDeleteReceive:
+  get:
+    x-resources: ["tenant", "v1-webhook"]
+    description: Get a webhook by its name
+    operationId: v1-webhook:get
+    parameters:
+      - description: The tenant id
+        in: path
+        name: tenant
+        required: true
+        schema:
+          type: string
+          format: uuid
+          minLength: 36
+          maxLength: 36
+      - description: The webhook name
+        in: path
+        name: v1-webhook
+        required: true
+        schema:
+          type: string
+    responses:
+      "200":
+        content:
+          application/json:
+            schema:
+              $ref: "../../../components/schemas/_index.yaml#/V1Webhook"
+        description: Successfully got the webhook
+      "400":
+        content:
+          application/json:
+            schema:
+              $ref: "../../../components/schemas/_index.yaml#/APIErrors"
+        description: A malformed or bad request
+      "403":
+        content:
+          application/json:
+            schema:
+              $ref: "../../../components/schemas/_index.yaml#/APIErrors"
+        description: Forbidden
+    summary: Get a webhook
+    tags:
+      - Webhook
+  delete:
+    x-resources: ["tenant", "v1-webhook"]
+    description: Delete a webhook
+    operationId: v1-webhook:delete
+    parameters:
+      - description: The tenant id
+        in: path
+        name: tenant
+        required: true
+        schema:
+          type: string
+          format: uuid
+          minLength: 36
+          maxLength: 36
+      - description: The name of the webhook to delete
+        in: path
+        name: v1-webhook
+        required: true
+        schema:
+          type: string
+    responses:
+      "200":
+        content:
+          application/json:
+            schema:
+              $ref: "../../../components/schemas/_index.yaml#/V1Webhook"
+        description: Successfully deleted webhook
+      "400":
+        content:
+          application/json:
+            schema:
+              $ref: "../../../components/schemas/_index.yaml#/APIErrors"
+        description: A malformed or bad request
+      "403":
+        content:
+          application/json:
+            schema:
+              $ref: "../../../components/schemas/_index.yaml#/APIErrors"
+        description: Forbidden
+      "404":
+        content:
+          application/json:
+            schema:
+              $ref: "../../../components/schemas/_index.yaml#/APIErrors"
+        description: Not found
+    tags:
+      - Webhook
+  post:
+    x-resources: ["tenant", "v1-webhook"]
```

Comment:
I think it may actually be better to remove `x-resources` from here and do all of the population at the handler level. That would get around all of the weird `maybeTenant` and casting and make it more straightforward.